purchase_stock_picking_return_invoicing/tests/test_purchase_stock_picking_return_invoicing.py

```diff
@@ -99,44 +118,34 @@ def test_purchase_stock_return_1(self):
         pick.button_validate()
         self.check_values(self.po_line, 0, 5, 0, 0, "to invoice")
         # Make invoice
-        inv_1 = (
-            self.env["account.invoice"]
-            .with_context(type="in_invoice")
-            .create(
-                {
-                    "partner_id": self.partner.id,
-                    "purchase_id": self.po.id,
-                    "account_id": self.partner.property_account_payable_id.id,
-                }
-            )
-        )
-        inv_1.purchase_order_change()
+        ctx = self.po.action_view_invoice()["context"]
+        active_model = self.env["account.move"].with_context(ctx)
+        view_id = "account.view_move_form"
+        with Form(active_model, view=view_id) as f:
+            f.partner_id = self.partner
+            f.purchase_id = self.po
+        inv_1 = f.save()
         self.check_values(self.po_line, 0, 5, 0, 5, "invoiced")
-        self.assertAlmostEqual(inv_1.amount_untaxed_signed, 50, 2)
+        self.assertAlmostEqual(inv_1.amount_untaxed_signed, -50, 2)
```

Comment:
@pedrobaeza this part, I need to change sign to pass test. As I seen in the code, it seem so. 
V12: https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_invoice.py#L71
V13: https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L1023